k8s/templates/studio-deployment.yaml

```diff
@@ -16,12 +16,36 @@ spec:
         app: {{ template "studio.fullname" . }}
         tier: app
     spec:
+      initContainers:
+      - name: collectstatic
+        image: {{ .Values.studioApp.imageName }}
+        workingDir: /contentcuration/
+        command:
+        - make
+        args:
+        - collectstatic
+        env:
+        - name: DJANGO_SETTINGS_MODULE
+          value: contentcuration.collectstatic_settings
+        - name: STATICFILES_DIR
+          value: /app/contentworkshop_static/
+        volumeMounts:
+        - mountPath: /app/contentworkshop_static/
+          name: staticfiles
       containers:
       - name: app
         image: {{ .Values.studioApp.imageName }}
+        workingDir: /contentcuration/contentcuration/
+        command:
+        - gunicorn
```

Comment:
I notice that in the current `make altprodserver` command, we also run these `ensurecrowdinclient downloadmessages compilemessages`. Are they not needed now? Thanks!

Reply:
Yup, no longer needed. We build the messages beforehand and commit them. Ref: [this file](https://github.com/learningequality/studio/blob/develop/contentcuration/locale/es/LC_MESSAGES/django.mo)

Reply:
ah I see. thanks for the explanation!